### PR TITLE
Core/Unit: Fixupfixup f8845ea.

### PR DESCRIPTION
**Pull request by robinsch**
Also chain related fixes.

### CODE REVIEW

File: src/game/Spell.cpp [hunk 1]
```diff
@@ -1689,18 +1692,36 @@ void Spell::SetTargetMap(SpellEffectIndex effIndex, uint32 targetMode, UnitList&
...
+                        jumpRadius = 7.5f; // Multi-Shot
+                        break;
+                    case SPELL_DAMAGE_CLASS_NONE:
+                    case SPELL_DAMAGE_CLASS_MAGIC:
+                        jumpRadius = 10.0f;
+                        break;
+                }
+
+                // chain lightning/heal spells and similar - allow to jump at larger distance and go out of los
+                bool isBouncingFar = (m_spellInfo->HasAttribute(SPELL_ATTR_EX4_AREA_TARGET_CHAIN)
```
Comment (DevDaemon) [inline src/game/Spell.cpp]: Automated build test failed here:
SPELL_ATTR_EX4_AREA_TARGET_CHAIN is undeclared.

Comment (robinsch) [inline src/game/Spell.cpp]: done

**Comment by DevDaemon** [comment]
Has conflicts.

**Comment by robinsch** [comment]
ci/circleci — Your tests passed on CircleCI!

**Comment by DevDaemon** [comment]
Still, git cannot merge.
"This branch has conflicts that must be resolved"

You have to merge with current repo (or rebase) and (force) update the pull request.

**Pull request closed without merging**